plutus-core-spec/plutus-core-specification.tex

```diff
@@ -856,7 +856,7 @@ \subsection{An abstract machine for evaluating Plutus Core programs}
 to evaluate an invalid program which should have been detected during
 parsing or typechecking.
 
-For efficiency reasons the behaviour of the machine defined in
+For efficiency reasons the behavior of the machine defined in
```

Comment:
Hmmm.  Being from the UK, I'd always spell it with a 'u'.  I suppose we're officially a US company though, so we should probably use US spelling.  I'd find that pretty painful though.  Maybe I should rephrase that to avoid the word.

Reply:
Rule 1 is consistency: this doc is in UK English so let's keep it that way for now. Maybe we should switch to US English in general.

Reply:
> Maybe we should switch to US English in general.

<sucks air through teeth>

Reply:
Actually I thought this document is in US english.

Reply:
I'm not sure how to show it one way or the other, but it was significantly written by British people so I'd expect it to be mostly UK English :laughing:

Reply:
But it has a lot of -ise verbs instead of -ize (e.g. finalise vs finalize) i thought the latter was british?

Reply:
Both versions are common in British English IME.

Reply:
I'm in the habit of writing -ise, but I can cope with -ize as well: they seem to be pretty interchangeable in the UK.   The main difference from my point of view is things like colour/color and neighbour/neighbor.

Reply:
I found https://en.wikipedia.org/wiki/Oxford_spelling and now I don't even know what I'm supposed to do.